packages/libraries/cli/README.md

```diff
@@ -34,6 +34,8 @@ curl -sSL https://graphql-hive.com/install.sh | sh
 
 <!-- commands -->
 
+- [`hive app:create FILE`](#hive-appcreate-file)
```

Comment:
Not sure we should call it `app:create`.
Maybe in some future we will use term `app` instead of `target` or `project`, or even have a dedicated features for apps.
I think we should be more explicit here and say it's the app deployment or something in that vibe.

Reply:
If we plan to use the name app for target and project, the term app deployment for this will also be confusing.